Kitodo/src/main/java/org/goobi/production/flow/statistics/hibernate/StatQuestProduction.java

```diff
@@ -57,53 +55,32 @@
     // default values set to days and volumesAndPages
     TimeUnit timeGrouping = TimeUnit.days;
     private CalculationUnit cu = CalculationUnit.volumesAndPages;
-    private static final Logger logger = LogManager.getLogger(StatQuestProduction.class);
 
     /**
-     * List objects here need to extend BaseDTO.
-     *
-     * <p>
-     * (non-Javadoc)
-     * </p>
-     *
-     * @see org.goobi.production.flow.statistics.IStatisticalQuestion#getDataTables(
-     *      List)
+     * @param dataSource
+     *            list of objects, needs to extend BaseDTO
      */
     @Override
     public List<DataTable> getDataTables(List<? extends BaseDTO> dataSource) {
-
-        // contains an intger representing "reihenfolge" in schritte, as defined
-        // for this request
-        // if not defined it will trigger a fall back on a different way of
-        // retrieving the statistical data
+        /**
+         * Contains an Integer representing step order, as defined for this
+         * request. If not defined, it will trigger a fallback on a different
+         * way of retrieving the statistical data.
+         */
         Integer exactStepDone = null;
-        String stepname = null;
 
         // gathering some information from the filter passed by dataSource
         // exactStepDone is very important ...
 
-        // TODO; find way to replace it
```

Comment:
This can't be remove until the proper way to restore this functionality is found.